operator_ui/src/components/Jobs/TaskListDag.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core/styles'
+import Typography from '@material-ui/core/Typography'
+import titleize from '../../utils/titleize'
+import * as d3dag from 'd3-dag'
+import * as d3 from 'd3'
+import { parseDot, Stratify } from './parseDot'
+
+const styles = ({ spacing }: Theme) =>
+  createStyles({
+    graph: {
+      padding: `${spacing.unit * 3}px 0px`,
```

Comment:
for one-offs could you just add the styles using `style={{ padding: `${spacing.unit * 3}px 0px` }}